libraries/llvm/rts.ll

```diff
@@ -367,39 +367,34 @@ define private %Reference @newReference(%Stack %stack) alwaysinline {
 
 define private %StackPointer @stackAllocate(%Stack %stack, i64 %n) {
     %stackPointer_pointer = getelementptr %StackValue, %Stack %stack, i64 0, i32 1, i32 0
+    %base_pointer = getelementptr %StackValue, %Stack %stack, i64 0, i32 1, i32 1
+    %limit_pointer = getelementptr %StackValue, %Stack %stack, i64 0, i32 1, i32 2
 
-    %stackMemory = getelementptr %StackValue, %Stack %stack, i64 0, i32 1
-    %memory = load %Memory, ptr %stackMemory
+    %currentStackPointer = load %StackPointer, ptr %stackPointer_pointer
+    %base = load %Base, ptr %base_pointer
```

Comment:
Does this impact performance? I assume LLVM optimizes this anyway, but I've deliberately left `base_pointer`, `limit_pointer`, `base`, etc. in the `realloc` branch since they're not needed in the more likely `continue` branch

Reply:
You are right, please move `base_pointer` and `base` to the `realloc` branch.